crates/client/assets/schema.sdl

```diff
@@ -868,9 +868,12 @@ type Policies {
 type PreconfirmationFailureStatus {
 	reason: String!
 	txPointer: TxPointer!
+	totalGas: U64!
+	totalFee: U64!
 	transactionId: TransactionId!
 	transaction: Transaction
 	receipts: [Receipt!]
```

Comment:
@xgreenx should we be concerned about the size of the `Preconfirmation` types? Receipts is an unbounded vec of Receipt. Could be full of logs. Also now we have outputs, etc.
cc: @rymnc

Reply:
It is fine, since the type is optional, if it becomes an issue, we always can stop sharing this kind of data to everyone